background.md

```diff
@@ -160,7 +160,7 @@ via the principles laid out in
 TUF.
 
 Verifiers can differentiate between the
-[old](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#42-file-formats-general-principles)
+[old](https://github.com/in-toto/specification/blob/master/in-toto-spec.md#42-general-principles)
```

Comment:
Could you update this link to point to https://github.com/in-toto/specification/blob/d416c1f334ac6b581f75c0fa65125fb434d7a610/in-toto-spec.md#42-file-formats-general-principles instead? The repository has since moved the old format out, and this is the previous release that still had the old envelope referred to here.

Reply:
fixed!